sentry/src/db/field/bignum.rs

```diff
@@ -0,0 +1,55 @@
+use std::error::Error;
+
+use bb8_postgres::tokio_postgres::types::{FromSql, IsNull, ToSql, Type};
+
+use bytes::BytesMut;
+use primitives::BigNum;
+
+#[derive(Debug)]
+pub(crate) struct BigNumPg(BigNum);
+
+impl Into<BigNum> for BigNumPg {
+    fn into(self) -> BigNum {
+        self.0
+    }
+}
+
+impl<'a> FromSql<'a> for BigNumPg {
```

Comment:
And since we can't `impl Trait` for types defined in another crate with Trait being a trait from another crate, we need this wrapper.